tools/rubygenerator/files/rules.cc

```diff
@@ -8,7 +8,7 @@
 #include "actions.hh"
 
 Rules::Rules(const rules::Options opt)
-    :!!rules_cc_inheritance!! sandbox_(opt.time)
+    : !!rules_cc_inheritance !!sandbox_(opt.time)
```

Comment:
This is a ~template. This will probably break the ruby generator.

Reply:
Nice catch!